instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/internal/ExporterConfigEvaluator.java

```diff
@@ -16,9 +16,17 @@ public final class ExporterConfigEvaluator {
 
   private ExporterConfigEvaluator() {}
 
+  /**
+   * Returns whether the exporter is enabled. This method is used for OTLP, logging and zipkin
+   * exporters.
+   */
   public static boolean isExporterEnabled(
       Environment environment, String exportersKey, String wantExporter, boolean defaultValue) {
 
+    if (environment.getProperty("otel.sdk.disabled", Boolean.class, false)) {
```

Comment:
```suggestion
    // the method is used to enable some Spring autoconfigurations
    if (environment.getProperty("otel.sdk.disabled", Boolean.class, false)) {
```

Reply:
In my opinion the word `some` is too vague, I have no idea what this comment means.

Reply:
@laurit better?

Reply:
Let's remove the comment if it is ambiguous

Reply:
@zeitlinger yes